packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -1043,7 +985,7 @@ def test_ip_per_task_app_with_unspecified_ip_address_DCOS_OSS_1366(
         """
         app = self._scheduler_alwaysthere_app()
 
-        # Remove the 'ipAddress' key completely, thereby triggering DCOS_OSS-1366.
+    # Remove the 'ipAddress' key completely, thereby triggering DCOS_OSS-1366.
         del(app["ipAddress"])
```

Comment:
Marathon should not be emitting `ipAddress` as part of the app spec, right? This test looks outdated